source/style-guide/style/parameters.txt

```diff
@@ -20,25 +18,13 @@ When documenting parameters, use the following guidelines:
 
    -  Write the first sentence of a description with an implied
       subject. For example, if the parameter is ``name``, the
-      description might be "Server name, which becomes the initial host
-      name of the server."
+      description might be "Initial host name of the server."
```

Comment:
**question:** Do we specify when to use the `.. example::` block vs. inline? I think it should be more than one line, use a block.

Reply:
Out of scope here, but [noted](https://docs.google.com/document/d/1gwg44VP4_P-hDn-QhLEDOpon-ZnzfEFLxmIa4qy1kgQ/edit).